src/dlgTriggerEditor.cpp

```diff
@@ -653,7 +653,15 @@ void dlgTriggerEditor::closeEvent(QCloseEvent *event)
 
 void dlgTriggerEditor::readSettings()
 {
-    QSettings settings("Mudlet", "Mudlet 1.0");
+    QSettings settings("mudlet", "Mudlet 1.0");
+
+    //Check if loaded. Otherwise, try to load from Mudlet directory instead of mudlet.
```

Comment:
I think you could write this comment better - the idea behind a comment is to explain something that isn't obvious. So in your case, anyone can see that you fallback to "Mudlet" after "mudlet" - but why do you do this? That's what you should explain, so write something about the bug and compatibility with older settings.